assets/base.css

```diff
@@ -1786,7 +1786,7 @@ details[open] > .share-button__fallback {
 .customer select {
```

Comment:
I activated the buy 2, get 1 50% off and I am not seeing the discount:

https://screenshot.click/31-14-n2gfl-s4h4r.png

(under discounts in admin)